CHANGELOG.md

```diff
@@ -29,6 +29,7 @@ Certbot adheres to [Semantic Versioning](https://semver.org/).
 * acme.standalone.BaseRequestHandlerWithLogging and
   acme.standalone.simple_tls_sni_01_server have been deprecated and will be
   removed in a future release of the library.
+* Certbot's config_changes subcommand has been removed
```

Comment:
Can you also note the functions that were removed like in https://github.com/certbot/certbot/pull/7198/files.